packages/core/src/textures/BaseTexture.ts

```diff
@@ -42,7 +43,7 @@ export interface BaseTexture extends GlobalMixins.BaseTexture, EventEmitter {}
  * @extends PIXI.utils.EventEmitter
  * @memberof PIXI
```

Comment:
webdoc doesn't have it currently, but type parameters will be documented with `@typeParam`. Let's add them right now!

```suggestion
 * @memberof PIXI
 * @typeParam T - The base-texture's resource.
 * @typeParam U - The resource parameters.
```